portfolio/src/main/webapp/script.js

```diff
@@ -12,31 +12,89 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+shownContent = "about-me";
+
+/* Displays the content title and text in the content container */
+function showContent(contentName){
+  document.getElementById(`${shownContent}-title`).style.display = "none";
```

Comment:
What you can do is wrap each of these pages in an `<article>` element.

```html
<div id="content-container">
  <!-- More articles -->

  <article id="education" class="hidden">
    <h3 class="content-title">Education</h3>
    <p class="content-text">Although I didn't always like school growing up,  
      I have also enjoyed learning new things about the world  
      and other people which pushed to start appreciating my  
      education more once I got to high school. The subjects I  
      liked the most were always math or science. I graduated   
      high school in summer 2019 and am heading into my  
      second year as a computer science major at the  
      University of Georgia.
  </article>

  <!-- More articles -->
</div>
```

Then you can use a generic handler for showing/hiding.

```javascript
/** Hides all other content and show element. */
function showContent(element) {
  const contentContainer = document.getElementById('content-container');

  Array.from(contentContainer.children).forEach(child => {
    child.classList.toggle('hidden', true);
  });

  element.classList.toggle('hidden', false);
}

window.addEventListener('DOMContentLoaded', () => {
  const navBar = document.getElementById('nav-bar');

  Array.from(navBar.children).forEach(child => {
    child.addEventListener('click', () => {
      const targetId = child.getAttribute('data-target-id');
      const targetElement = document.getElementById(targetId);
      showContent(targetElement);
    });
  });
});
```

Add `data-target-id` to the nav item

```html<li id="education">Education</li>
<li data-target-id="eduction">Education</li>
```

Reply:
Oh wow that makes it a lot more neat, thanks!